public/app/percona/inventory/Inventory.constants.ts

```diff
@@ -125,3 +125,7 @@ export const PAGE_MODEL: PageModel = {
   id: 'inventory',
   children: PAGE_TABS.map(({ title, id, path }) => ({ title, id, path })),
 };
+
+export const GET_SERVICES_TOKEN = 'getServices';
+export const GET_NODES_TOKEN = 'getNodes';
+export const GET_AGENTS_TOKEN = 'getAgents';
```

Comment:
a more descriptive name would be better, maybe something like `GET_SERVICES_CANCEL_TOKEN`

Reply:
👍